eventuals/event-loop.cc

```diff
@@ -57,7 +57,7 @@ void EventLoop::Clock::Resume() {
 
 ////////////////////////////////////////////////////////////////////////
 
-void EventLoop::Clock::Advance(const std::chrono::nanoseconds& nanoseconds) {
+void EventLoop::Clock::Advance(std::chrono::nanoseconds&& nanoseconds) {
```

Comment:
Doesn't seem like we need to change this one though given we don't need to copy or move so let's stick with `const &` so we simplify callers.